quail/processes/wps_climdex_su.py

```diff
@@ -0,0 +1,149 @@
+from rpy2 import robjects
+from pywps import Process, LiteralInput, ComplexInput, ComplexOutput, FORMATS
+from pywps.app.Common import Metadata
+from pywps.inout.formats import Format
+from tempfile import NamedTemporaryFile
+
+from wps_tools.utils import log_handler, collect_args, common_status_percentages
+from wps_tools.io import log_level
+from quail.utils import get_package, logger
+
+
+class ClimdexSU(Process):
+    """
+    Takes a climdexInput object as input and computes the SU
+    (summer days) climdexindex:  that is,  the annual count of days where
+    daily maximum temperature exceeds 25 degreesCelsius
+    """
+
+    def __init__(self):
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{
+                "build_rdata": 90,
+            },
+        )
+        inputs = [
+            ComplexInput(
+                "climdex_input",
+                "climdexInput",
+                abstract="Rdata (.rda) file containing R Object of type climdexInput",
+                supported_formats=[
+                    Format("application/x-gzip", extension=".rda", encoding="base64")
+                ],
+            ),
+            LiteralInput(
+                "ci_name",
+                "climdexInput name",
+                abstract="Name of the climdexInput obejct",
+                min_occurs=1,
+                max_occurs=1,
+                data_type="string",
+            ),
```

Comment:
Since we now have an input `ci_name`, I think we also need to have an input for the output object name as well unless we have any better strategies for R object I/O.